modules/masterbar/masterbar/masterbar.php

```diff
@@ -164,7 +170,7 @@ class_exists( 'Jetpack_AMP_Support' )
 
 		if ( Jetpack::is_module_active( 'notes' ) && $this->is_rtl() ) {
 			// Override Notification module to include RTL styles.
-			add_action( 'a8c_wpcom_masterbar_enqueue_rtl_notification_styles', '__return_true' );
+			add_action( '_enqueue_rtl_notification_styles', '__return_true' );
```

Comment:
I'm not sure on the deal here. Seems PHPCS automatically "fixed" this for me but is it right? Surely we don't want to change the action?

Reply:
Note this happens in _several_ places in this file.

Reply:
If you put the prefix back, does it get removed again?

Reply:
Fixed.